src/qt/walletmodel.cpp

```diff
@@ -342,7 +342,7 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(WalletModelTransaction &tran
         transaction_array.append(&(ssTx[0]), ssTx.size());
     }
 
-    // Add addresses / update labels that we've sent to to the address book,
+    // Add addresses / update labels that we've sent to the address book,
```

Comment:
`Add or update <addresses/labels that we've sent to> to <the address book>` - I don't think this is a correct fix.

Maybe:

```
// Add or update output addresses and labels to the address book,
```